modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -272,24 +274,51 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
-	@ignore = "Test Stub"
 	@priority = 5
 	test UntoggledSettingsAreAppliedImmediately {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("And underline class is applied to body element") {
+			AccessibilityMenu.openAccessibilityMenu();
 
-		// // Given home page
-		// // And underline class is applied to body element
-		// // // verify show underline config in accessibility menu is on
-		// // // verify body element does have "c-prefers-link-underline" class present
-		// // When access Accessibility Menu
-		// // And toggle OFF show underline effect in links
-		// // Then Powered by Liferay link does not have underline class applied
-		// // // assert body element does not have "c-prefers-link-underline" class present
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
 
-		//}
+		task ("verify show underline config in accessibility menu is on") {
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
 
+		task ("verify body element does have 'c-prefers-link-underline' class present") {
+			var key_class = ${key_class};
+
+			AssertElementPresent(
+				key_class = "c-prefers-link-underline",
+				locator1 = "AccessibilityMenu#SELECTED_CLASS");
+		}
```

Comment:
Keep the poshi steps, remove the task descriptions. When we create the test scenarios please don't add in the comments. The comments are just clues for testers in translating the Given When Then steps to poshi. Please only use Given When Then format for structuring tasks.

Reply:
remember to do this